src/api/constants/index.ts

```diff
@@ -116,7 +116,7 @@ export const ACCOUNT = {
       STRING_ENCODING: 'ascii' as BufferEncoding,
     },
     TOKEN: {
-      EXPIRY: '8h',
+      EXPIRY: '12h',
```

Comment:
Do we need a ticket for this?

Reply:
Yes please

Reply:
Have created a task here and added updated the PR title https://ukef-dtfs.atlassian.net/browse/EMS-2194